src/start_server.ts

```diff
@@ -143,6 +152,23 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxyRoot && options.proxyTarget) {
+    const apiProxy = httpProxy.createProxyServer();
+    const proxyHandler = (req: express.Request, res: express.Response) => {
+      // Remove proxied part from the url
+      req.url = req.url.slice(options.proxyRoot.length + 1);
```

Comment:
It might be good to have an assertion that the request path starts with the proxy root to guard against cases where express unexpectedly matches something else based on special characters in the proxy root.